bedrock/mozorg/templates/mozorg/contact/spaces/berlin.html

```diff
@@ -47,6 +47,9 @@ <h2 itemprop="name" class="fn org">{{ _('<span>Mozilla</span> Berlin') }}</h2>
       {% trans link_sumo='https://support.mozilla.org/' %}
         Need help with Firefox? <a href="{{ link_sumo }}" class="go">Get it on our support site</a>
       {% endtrans %}
+      {% trans link_sumo='https://careers.mozilla.org/listings/?location=Berlin' %}
```

Comment:
nit: maybe we should name this variable something else because it is not a sumo link. Sumo stands for [support.mozilla.org](https://support.mozilla.org/). Perhaps, `link_careers`?

Reply:
Okay I will do it. Thanks